openpype/hosts/traypublisher/addon.py

```diff
@@ -58,3 +59,64 @@ def launch():
     from openpype.tools import traypublisher
 
     traypublisher.main()
+
+
+@cli_main.command()
+@click_wrap.option(
+    "--csv-filepath",
+    help="Full path to CSV file with data",
+    type=click.Path(exists=True),
```

Comment:
`Path` seems to be lacking in the `click_wrap` wrapper @iLLiCiTiT ? :)

Reply:
We have discussed the issue with @iLLiCiTiT and the `click_wrap` wasn't intended to be going that deep into feature set. So the code is using wrapper now but also its own Path checking. Look here for more info [ayon-core #100](https://github.com/ynput/ayon-core/pull/100/files#diff-677da50ac9e75faa8d39432055a28047022163441877a82413cfcea2382e7163R110-R111)